mir_perception/mir_object_recognition/ros/scripts/rgb_object_recognizer_node

```diff
@@ -149,7 +149,7 @@ class RGBObjectRecognizer():
                             result.roi = roi
                             objects.append(result)
                         if self.debug:
-                            print("DETECTED OBJECTS: ", labels)
+                            rospy.logdebug("DETECTED OBJECTS: ", labels)
```

Comment:
You need the string formatting operator: `rospy.logdebug('Detected objects: %s', labels)`

Reply:
resolved